scripts/release-config.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+GITHUB_DEST_BRANCH_NAME=$(echo ${CODEBUILD_WEBHOOK_BASE_REF} | cut -d "/" -f3-)
+GIT_COMMIT_SHA=$(git rev-parse $GITHUB_DEST_BRANCH_NAME)
+GIT_COMMIT_SHORT_SHA=$(git rev-parse --short=8 $GITHUB_DEST_BRANCH_NAME)
+RELEASE_DATE=$(date +'%Y%m%d')
+AGENT_VERSION=$(cat ../VERSION)
+
+cat << EOF > agent.json
+{
+    "agentReleaseVersion" : "$AGENT_VERSION",
+    "releaseDate" : "$RELEASE_DATE",
+    "initStagingConfig": {
+    "release": "1"
+    },
+    "agentStagingConfig": {
+    "releaseGitSha": "$GIT_COMMIT_SHA",
+    "releaseGitShortSha": "$GIT_COMMIT_SHORT_SHA",
+    "gitFarmRepoName": "MadisonContainerAgentExternal",
+    "gitHubRepoName": "aws/amazon-ecs-agent",
+    "gitFarmStageBranch": "v${AGENT_VERSION}-stage",
+    "githubReleaseUrl": ""
+    }
+}
+EOF
+
+# Downloading the agentVersionV2-<branch>.json (it is assumed that it already exists)
+aws s3 cp ${RESULTS_BUCKET_URI}/agentVersionV2/agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json ./agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json
+echo "Downloaded agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json"
+cat agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json
+
+# Grabbing the new current and release agent version
+CURR_VER=$(jq -r '.agentReleaseVersion' agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json)
+
+# Checking to see if there's a new release agent version to be released
+if [[ ! $AGENT_VERSION =~ "${CURR_VER}" ]] ; then
+    # Updating the agentVersionV2-<branch>.json file with new current and release agent versions and copying it to a temp file
+    cat <<< $(jq '.agentReleaseVersion = '\"$AGENT_VERSION\"' | .agentCurrentVersion = '\"$CURR_VER\"'' agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json) > agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}-COPY.json
```

Comment:
Is this command the same as 
```
jq ".agentReleaseVersion = \"$AGENT_VERSION\" | .agentCurrentVersion = \"$CURR_VER\"" agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}.json > agentVersionV2-${GITHUB_SOURCE_BRANCH_NAME}-COPY.json
```
?

If so, I think the version in this comment reads better?

Reply:
Thanks!